package.json

```diff
@@ -34,6 +34,7 @@
     "chai": "^3.0.0",
     "eslint": "^0.24.0",
     "eslint-config-airbnb": "0.0.6",
+    "escope": "3.3.0",
```

Comment:
Not sure if I'm remembering correctly, but I think I needed to add that to bypass an issue with babel-eslint back when redux-actions was on babel 5. I'll remove it now.
https://github.com/estools/escope/issues/99